recipes/napari-cursor-tracker/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "napari-cursor-tracker" %}
+{% set version = "0.1.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/napari-cursor-tracker-{{ version }}.tar.gz
+  sha256: ddf189aef636fb2308d6d617872607d1c8a241abe3a4ebac6506eaa5a7e571cb
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.8
+    - setuptools >=64
+    - setuptools-scm
+    - wheel
+  run:
+    - magicgui
+    - napari
+    - numpy
+    - python >=3.8
+    - qtpy
+
+test:
+  imports:
+    - napari_cursor_tracker
+  commands:
+    - pip list
+    - pip check
+  requires:
+    - pip
+    - pyqt
```

Comment:
Same qt issue again. I have probably overlooked this in previous reviews but we should fix this. In contrast to pip we have the means with conda and thus should always provide the user with a fully functional setup.

Reply:
@xhochy again, same comment from previous recipe (and this is something that has been  decided for napari). Napari by default does not provide a binding also on pip, so a user needs to install napari and then a binding, pyside or pyqt, hence why all these plugins will need to include a binding so the imports or tests pass.

This is not something we need to fix. Is a packaging decision. See https://napari.org/stable/tutorials/fundamentals/installation.html

Reply:
The variant option already discussed and voted against.

See:
- https://github.com/conda-forge/napari-feedstock/pull/48
- https://github.com/conda-forge/napari-feedstock/issues/47

The napari team is ok with having this as is for both conda and pip.

Reply:
Fine with me then.